exercises/macros/example.rs

```diff
@@ -1,5 +1,5 @@
 // Ignoring the README's injunction, this is heavily based on the implementation from maplit.
-// Original source is at https://github.com/bluss/maplit/blob/master/src/lib.rs#L27-L60
+// Original source is at https://github.com/bluss/maplit/blob/main/src/lib.rs#L27-L60
```

Comment:
```suggestion
// Original source is at https://github.com/bluss/maplit/blob/master/src/lib.rs#L27-L60
```

`bluss/maplit` retains `master` as primary branch; checked now.